request.go

```diff
@@ -395,6 +396,22 @@ func (r *Request) SetAuthToken(token string) *Request {
 	return r
 }
 
+// SetAuthScheme method sets the auth token scheme type in the HTTP request. For Example:
```

Comment:
@rigrassm I think its good to include the RFC and IANA links with few texts in the godoc would be helpful for the user.

Reply:
Added in both client.go and request.go